onnx_mxnet/tests/test_layers.py

```diff
@@ -216,5 +216,14 @@ def test_softmax(self):
         numpy_op = exp_score / exp_score.sum(0)
         npt.assert_almost_equal(output, numpy_op)
 
+    def test_squeeze(self):
+        """Test for squeeze operator"""
+        node_def = helper.make_node("Squeeze", ["ip1"], ["op1"], axes=[1, 3])
+        ip1_shape = (3, 1, 2, 1, 4)
+        ip1 = np.random.ranf(ip1_shape).astype("float32")
```

Comment:
Call _random_array() function, which is defined in the same file, passing "ip1_shape" as input to do the same.